### DISCUSSION TITLE
Buying & Selling Single Cards ? [How To]

### DISCUSSION THREAD

**Discussion by SENPAY98K** [Refloow/Steam-Card-Bot-PRO#17] [Q&A] [answered]
Is there a way to buy and buy single cards for a specific price? for example the sale event cards, buying any card of the set for 90 gems and selling for 100.

**Comment by Refloow** [comment, marked as answer]
**Sorry, but short answer is no there is not.**
_Script doesn't have any features that would allow you to do that normally._

One dumb way to accomplish that would be removing bots database and inside of the utils.js file fill the games you would want to buy the cards of, but in the game data instead of number of cards per set, you could in theory set it to 1. Disable all buy commands in the config that would allow users to buy , and set that script to work on a separate account. Change the pricing, and then script would buy single cards but think its full "sets". You can also enable or disable commands, so for example you could only use gems or tf2 metal.

`
I wouldn't recommend doing it that way since i haven't done it myself but in theory that could work, even tho it will be weird, and can cause some glitches.`

**I'm working on a script that would allow you to buy single cards, emoticons, backgrounds and booster packs. Cant promise when will be published since there is not many people asking for it, and we have some more projects planned.**

_We will most likely setup and community donation goals, so people can vote that way what they want us to build next :D_
